pkg/kubelet/lifecycle/handlers.go

```diff
@@ -55,9 +58,16 @@ func (hr *HandlerRunner) Run(containerID kubecontainer.ContainerID, pod *api.Pod
 		var buffer bytes.Buffer
 		output := ioutils.WriteCloserWrapper(&buffer)
 		err := hr.commandRunner.ExecInContainer(containerID, handler.Exec.Command, nil, output, output, false)
+		if err != nil {
+			glog.V(1).Infof("Lifecycle-Probe for Container %q in Pod %q failed - %q", container.Name, pod.Name, buffer.String())
```

Comment:
Done & squashed

On Fri, May 20, 2016 at 4:10 PM, Yu-Ju Hong <notifications@github.com>
wrote:

> In pkg/kubelet/lifecycle/handlers.go
> <https://github.com/kubernetes/kubernetes/pull/25681#discussion_r64119307>
> :
>
> > @@ -55,9 +58,16 @@ func (hr *HandlerRunner) Run(containerID kubecontainer.ContainerID, pod *api.Pod
> >  		var buffer bytes.Buffer
> >  		output := ioutils.WriteCloserWrapper(&buffer)
> >  		err := hr.commandRunner.ExecInContainer(containerID, handler.Exec.Command, nil, output, output, false)
> > +		if err != nil {
> > +			glog.V(1).Infof("Lifecycle-Probe for Container %q in Pod %q failed - %q", container.Name, pod.Name, buffer.String())
>
> How about "printing the handler or command"? I think you missed this one :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/25681/files/b7721421008b473c7dd227ec3dab52a15e099332#r64119307>
>